packages/backend/src/modules/tracked-txs/modules/l2-costs/BlobPriceProvider.ts

```
@@ -0,0 +1,88 @@
+import type { Logger } from '@l2beat/backend-tools'
+import type { IRpcClient } from '@l2beat/shared'
+
+export class BlobPriceProvider {
+  private readonly maxBlocksPerRequest = 1000
+  constructor(
+    private readonly logger: Logger,
+    private readonly ethRpcClient: IRpcClient,
+  ) {
+    this.logger = this.logger.for(this)
+  }
+
+  async getBlobPricesByBlockRange(
+    blockRange: [number, number],
+  ): Promise<Map<number, bigint>> {
+    const [oldestBlock, newestBlock] = blockRange
+
+    if (oldestBlock > newestBlock) {
```

I think we should just throw an Error instead of returing valid response